### ISSUE TITLE
Unable to make Live Update work in and Capacitor application with Appflow

### ISSUE THREAD

**Issue by mvaljento** [ionic-team/capacitor#6384]
## Bug Report

Not sure if this as Capacitor, Appflow or Ionic issue, or perhaps I have understood the concept of Live Update wrong. In that case, feel free to correct me.

### Capacitor Version

Latest Dependencies:

  @capacitor/cli: 4.7.0
  @capacitor/core: 4.7.0
  @capacitor/android: 4.7.0
  @capacitor/ios: 4.7.0

Installed Dependencies:

  @capacitor/cli: 4.7.0
  @capacitor/core: 4.7.0
  @capacitor/android: 4.7.0
  @capacitor/ios: 4.7.0

### Platform(s)

Android 13

### Current Behavior

I have followed the Appflow Live Update setup instructions and have the following content in android/app/src/main/res/values/strings.xml. I have build an Angular Capacitor application in Appflow and deployed the resulting APK file from Firebase to a device.

`<?xml version='1.0' encoding='utf-8'?>
<resources>
    ...
    <string name="ionic_channel_name">Alpha</string>
    <string name="ionic_update_method">background</string>
    <string name="ionic_max_versions">2</string>
    <string name="ionic_min_background_duration">10</string>
    <string name="ionic_update_api">https://api.ionicjs.com</string>
</resources>
`

When building a JS release with a change to the Angular application and deploying a live update to "Alpha" channel, I expect the deployed application to be updated at some point automatically on the device. However nothing happens and the app does not get updated.

### Other Technical Details

`npm --version` output:
9.1.1

`node --version` output:
v16.15.0

### Additional Context

"@capacitor/app": "^4.1.1",
"cordova-plugin-ionic": "5.5.3"

**Comment by jcesarmobile** [comment]
background mode will check for updates on launch and apply them on next launch

Are you changing html/css/js? because the commit says "update version number for testing", if you changed the app version number, that's a native change, not a web change, so there won't be any changes to update, the live update is only possible for web code, native changes always require new app installs.

**Comment by mvaljento** [comment]
@jcesarmobile No, that's just a value in the Angular environment. Nothing to do with the native version number. Does the Live Update need to be separately enabled somewhere in addition to the strings.xml options I described in my original post?

**Comment by mvaljento** [comment]
Okay, I found the reason: when I switched from Cordova to Capacitor, I had copied over some configuration which disabled the live updates, not only for local development, but also for live deployment. I removed that and now the live update is working!

```
cordova: {
    preferences: {
      DisableDeploy: "false",
    },
  },
```
